node-api/handlers/beacon/types/response.go

```diff
@@ -30,15 +30,26 @@ type GenericResponse struct {
 	Data                any  `json:"data"`
 }
 
-type ValidatorResponse struct {
+type Response struct {
```

Comment:
This `Response` type is identical to `GenericResponse` above. Why do we need both?

Reply:
Would agree. I think we can just return GenericResponse from `NewResponse(data any)` and get rid of the `ValidatorResponse` struct

Reply:
My bad, addressed.